woocommerce/woocommerce-android
# [Mobile Payments] Add tracks event when onboarding check was successful

Closes: #9769


### Description

The PR:
* Changes the tracking of the onboarding result - now it's tracked right after the status received
* Adds tracking of the `CARD_PRESENT_ONBOARDING_COMPLETED` event

### Testing instructions

* On the site where IPP onboarding is complete -> more -> payments -> notice:

```
card_present_onboarding_completed, Properties: {"plugin_slug":"woocommerce-payments","country":"US","blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","is_debug":true,"site_url":"https:\/\/site-for-woocommerce12a3fasdf45dfs6789.mystagingwebsite.com"}
```

* On the site where IPP onboarding is not complete -> more -> payments -> notice:
```
Tracked: card_present_onboarding_not_completed, Properties: {"reason":"wcpay_in_test_mode_with_live_account","plugin_slug":"woocommerce-payments","country":"US","blog_id":198219640,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/anotherpaymentssite.wpcomstaging.com"}
```
